core/ui/src/components/settings/SyslogConfigureModal.vue

```diff
@@ -0,0 +1,369 @@
+      this.time = "";
+      this.last_timestamp_formatted = "";
+    },
+    onModalShown() {
+      if (
+        this.configuration.status == "active" ||
+        this.configuration.status == "failed"
+      ) {
+        this.toggleEnabled = true;
+      }
+      if (this.configuration.address != "") {
+        this.address = this.configuration.address;
+      }
+      if (this.configuration.port != "") {
+        this.port = this.configuration.port;
+      }
+      if (this.configuration.protocol != "") {
+        this.protocolVal = this.configuration.protocol;
+      }
+      if (this.configuration.format != "") {
+        this.formatVal = this.configuration.format;
+      }
+      if (this.configuration.last_timestamp == "") {
+        this.dateVal = "choose_date_time";
+      } else {
+        const last_timestamp_date = new Date(this.configuration.last_timestamp);
+        this.last_timestamp_formatted = this.formatDate(
+          last_timestamp_date,
+          "Pp"
+        );
+      }
+      this.date = this.formatDate(new Date(), "yyyy-MM-dd");
+      this.time = "00:00";
+    },
+    validateSyslogInput() {
+      this.clearErrors(this);
+      let isValidationOk = true;
+
+      if (!this.address) {
+        this.error.address = this.$t("common.required");
+        isValidationOk = false;
+      }
+
+      if (!this.port) {
+        this.error.port = this.$t("common.required");
+        isValidationOk = false;
+      }
+      if (this.port && isNaN(this.port)) {
+        this.error.port = this.$t("system_logs.loki.valid_number");
+        isValidationOk = false;
+      }
+      if (this.port && this.port <= 0) {
+        this.error.port = this.$t("system_logs.loki.greater_than_0");
+        isValidationOk = false;
+      }
+
+      return isValidationOk;
+    },
+    async setSyslog() {
+      if (!this.validateSyslogInput()) {
+        return;
+      }
+
+      this.loading.setSyslog = true;
+      this.error.setSyslog = "";
+      const taskAction = "set-syslog-forwarder";
+
+      // register to task abortion
+      this.$root.$once(taskAction + "-aborted", this.setSyslogAborted);
+
+      // register to task completion
+      this.$root.$once(taskAction + "-completed", this.setSyslogCompleted);
+
+      if (this.dateVal == "choose_date_time") {
+        this.start_time = new Date("".concat(this.date, " ", this.time));
+      } else {
+        this.start_time = "";
+      }
+
+      if (this.toggleEnabled) {
+        this.active = true;
+      } else {
+        this.active = false;
+      }
+
+      const res = await to(
+        this.createModuleTaskForApp(this.lokiToEdit.instance_id, {
+          action: taskAction,
+          data: {
+            active: this.active,
+            address: this.address,
+            port: this.port,
+            protocol: this.protocolVal,
+            format: this.formatVal,
+            start_time: this.start_time,
+          },
+          extra: {
+            isNotificationHidden: true,
```

Comment:
`isNotificationHidden: true` should be used only for read actions. Use this as reference: https://github.com/NethServer/ns8-core/blob/main/core/ui/src/components/nodes/SetFqdnModal.vue#L218-L220